unity-renderer/Assets/Scripts/Tests/CursorControllerTests/CursorControllerTests.cs

```diff
@@ -220,6 +220,8 @@ public IEnumerator FeedbackIsNotDisplayedOnParent()
             DCLCharacterController.i.SetPosition(new Vector3(3, 2, 12));
             cameraController.SetRotation(0, 0, 0, new Vector3(0, 0, -1));
 
+            yield return null;
+            yield return null;
```

Comment:
This smells fishy, why do we need to wait for 3 frames?

Reply:
I don't remember exactly, but IIRC it was a race condition where sometimes it took more than 1 frame to change the cursor sprite, so I set two more nulls to ensure that the race condition doesn't happen anymore

Reply:
I will add a comment for the next time to this kind of approach so we can see in the future why are we doing this